lib/host/wasi/inode-linux.cpp

```diff
@@ -1011,7 +1011,7 @@ WasiExpect<void> INode::sockRecv(Span<Span<uint8_t>> RiData,
 WasiExpect<void> INode::sockSend(Span<Span<const uint8_t>> SiData,
                                  __wasi_siflags_t,
                                  __wasi_size_t &NWritten) const noexcept {
-  int SysSiFlags = 0;
+  int SysSiFlags = MSG_NOSIGNAL;
```

Comment:
You are changing the default value of the `SiFlags`. Will this break the previous behaviour?

Reply:
When the server shuts down, the client will exit on send without MSG_NOSIGNAL